src/libAtomVM/bif.c

```diff
@@ -39,9 +39,12 @@
 #include "bifs_hash.h"
 #pragma GCC diagnostic pop
 
-#define RAISE_ERROR(error_type_atom) \
-    ctx->x[0] = ERROR_ATOM;          \
-    ctx->x[1] = (error_type_atom);   \
+#define RAISE_ERROR(error_type_atom)                                     \
+    if (error_type_atom == OUT_OF_MEMORY_ATOM) {                         \
+        fprintf(stderr, "Out of memory at %s:%i\n", __FILE__, __LINE__); \
```

Comment:
This macro is called several times, this change would increase the amount of generated code.
I'd rather handle this special case elsewhere.